docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:latest
```

Comment:
Freeze the version to the version that you are developing to avoid possible breaking changes of the language in newer versions.

Reply:
@raonismaneoto was this comment already addressed? if so, resolve the conversation